manifests/params.pp

```diff
@@ -44,7 +44,7 @@
   # and greping for "puppet enterprise". With Puppet 4 and PE 2015.2, there
   # is no longer a "PE Puppet", and so that fact will no longer work.
   # Instead check for the `is_pe` fact or if a PE provided function is available
-  $_is_pe = (getvar('::is_pe') or is_function_available('pe_compiling_server_version'))
+  $_is_pe = (getvar('::is_pe') or stdlib::has_function('pe_compiling_server_version'))
```

Comment:
I'm fine with this, but we need to keep in mind that it now requires stdlib 9.3. with my `defined()` it works on all stdlib versions

Reply:
Yes, I am not happy with this neither… :sob:

I am currently fighting with bolt to use `apply` with modules that support stdlib ">= 9" but Bolt's `apply` rely on this module which currently only support stdlib "< 9".

Reply:
FYI I rolled out my code with `defined()` at a few customers and that works fine as expected

Reply:
It is fragile but avoids the need for bumping the minimum version of stdlib.  I have no PE anywhere around, so I am not really concerned about this fragility, if it is okay, then it is fine!  I'll rework the commits I added to take this into account. If it fix CI and you can test this in the field to validate the change, I think we will be good to go.

Reply:
> I'll rework the commits I added to take this into account. If it fix CI and you can test this in the field to validate the change, I think we will be good to go.

Code pushed. CI failures are related to my commit messages on the commits that need merging, so if you can confirm it fix your issue on PE, you can merge the commits (fixup/squash) and I think CI will be all green :grin: